### PR TITLE
Amarachi 12

### PR DESCRIPTION
**Pull request by Amarachi9**
-  I Added changeUserEmail

### CODE REVIEW

File: src/controllers/userController.js [hunk 1]
```diff
@@ -77,6 +77,10 @@ const user = {
         userModel.findOne({_id: req.params.id}).then(user => res.json({status: 'Success', message: 'User phone number', data: user.phone}))
         .catch((err) => res.status(400).json({status: 'Failed', message: 'An Error Occurred', data: null}));
     }, 
+    changeUserEmail: (req, res) =>{
+        userModel.findOne({_id: req.params.id}).then(user => res.json({status: 'Success', message: 'User email', data: user.email}))
```
Comment (goodhands) [inline src/controllers/userController.js:81]: Please change message to **User email updated successfully**

**Comment by Amarachi9** [comment]
Changed message in changeUserEmail to User email updated successfully

File: src/controllers/userController.js (hunk 1 above)
Comment (goodhands) [reply src/controllers/userController.js]: Also, resolve your merge conflicts, change `changeUserEmail` to `setUserEmail`, let's keep the same naming convention.

**Comment by goodhands** [comment]
> Changed message in changeUserEmail to User email updated successfully

You still have merge conflicts

**Comment by Amarachi9** [comment]
Changed changeUserEmail to setUserEmail

**Pull request closed without merging**